lib/Executions.js

```diff
@@ -22,6 +21,41 @@ class Executions {
 
     return result
   };
+
+  /**
+   * Returns a list of executions summary, with filtering.
+   *
+   * @returns {Promise.<Execution[]>} The returned objects include the execution summary,
+   *                                  detailed objects with data about the execution.
+   */
+  async getExecutions () {
+    return this.requestManager.request({
+      actionUrl: this.executionsActionUrl + '?pageSize=1000&pageNum=1',
+      method: 'GET'
+    })
+  }
 }
 
 module.exports = Executions
+
+/**
+ * @typedef {Object} Execution
```

Comment:
this for typescript or something?

Reply:
Yes, it is a JSdoc type definition, something like an interface in Typescript (if we have no Typescript) :)